composer.json

```diff
@@ -29,6 +29,7 @@
         "doctrine/orm": "^2.3",
         "friendsofphp/php-cs-fixer": "^2.8",
         "friendsoftwig/twigcs": "^3.1.2",
+        "symfony/expression-language": "^4.4@dev",
```

Comment:
Use `^4.4|$5.0`

With the config you have here, the tests will always run against 4.4@dev. But we would prefer that our "lowest deps" test (with PHP 7.1) uses 4.4 stable and our other tests (using PHP 7.4) use the latest 5.* stable release.

Reply:
Done.